Library/Formula/global.rb

```diff
@@ -44,6 +45,10 @@ def install
       args << "--with-exuberant-ctags=#{Formula["ctags"].opt_bin}/ctags"
     end
 
+    if build.with? "sqlite3"
+      args << "--with-sqlite3"
```

Comment:
` args << "--with-sqlite3" if build.with? "sqlite3" ` - No need to wrap it in a block if it's the only thing.